torch/distributed/tensor/_dispatch.py

```diff
@@ -34,6 +35,23 @@
 logger = logging.getLogger(__name__)
 
 
+def as_strided_handler(
+    op_call: torch._ops.OpOverload,
+    args: tuple[object, ...],
+    kwargs: dict[str, object],
+):
+    args, kwargs = fill_defaults(op_call._schema, args, kwargs)
+    assert not kwargs
+    tensor, size, stride, storage_offset = args
+    if (
+        tensor.size() == tuple(size)
+        and tensor.stride() == tuple(stride)
+        and (storage_offset is None or tensor.storage_offset() == storage_offset)
+    ):
+        return torch.ops.aten.alias.default(tensor)
```

Comment:
what does alias do on a dtensor? is the .local_tensor on the alias gauranteed to be an alias of the local_tensor on the original, and we guarantee all mutating operations on the alias/original affect each other?